servers/physics/joints/cone_twist_joint_sw.cpp

```diff
@@ -107,6 +107,7 @@ ConeTwistJointSW::ConeTwistJointSW(BodySW *rbA, BodySW *rbB, const Transform &rb
 	B->add_constraint(this, 1);
 
 	m_appliedImpulse = 0;
+	m_angularOnly = 0;
```

Comment:
That works, but `false` would be better. Maybe also moved up next to `m_solveTwistLimit` to match the header: https://github.com/dotquixote/godot/blob/051146decf897b55fabc894ede8371d5cb86c538/servers/physics/joints/cone_twist_joint_sw.h#L100